src/test/resources/snapshots/com/conveyal/datatools/manager/jobs/DeployJobTest/canMakeGraphBuildAndServeManifestAndUserData-0.json

```diff
@@ -1,9 +1,12 @@
 {
   "baseFolder" : "/var/otp/graphs",
-  "baseFolderDownloads" : [ "s3://datatools-dev/test-deploy/build-config.json", "s3://datatools-dev/test-deploy/router-config.json" ],
+  "baseFolderDownloads" : [ {
+    "uri" : "s3://datatools-dev/test-deploy/build-config.json"
+  }, {
+    "uri" : "s3://datatools-dev/test-deploy/router-config.json"
+  } ],
   "buildGraph" : true,
-  "graphObjUrl" : "s3://datatools-dev/test-deploy/Graph.obj",
-  "gtfsDownloads" : [ ],
+  "graphObjUri" : "s3://datatools-dev/test-deploy/Graph.obj",
   "jarFile" : "/opt/otp/otp-latest-trimet-dev",
   "jarUrl" : "https://opentripplanner-builds.s3.amazonaws.com/otp-latest-trimet-dev.jar",
```

Comment:
`jarUrl` should be `jarUri` now.

Reply:
Fixed in https://github.com/ibi-group/datatools-server/pull/363/commits/d30a9edb8db1fe4acf1bc521ecae471938349f05